english/jacket/marshmallow-jacket.md

```diff
@@ -3,3 +3,4 @@ creating a new jacket out of marshmallows. Your hope is to make this one less re
 You get started right away melting and cooling marshmallows in to the form of a jacket.
 
 Do you:
+Give up and have [milk and cookies](../milk-and-cookies/jacket-cookie.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.